release_notes.md

```diff
@@ -5,3 +5,38 @@
 The file describes all the changes, that are affected the usage of the application.
 
 The format is based on [Release_notes format](https://github.com/ios-course/ironfoudation-team-project/wiki/Release_notes-format).
+
+## [0.1.0] - [29.05.2023]
+
+### New features
+
+`Quicklook`: Users can check a list of pet sitters available for pet owners. Each list row includes an avatar, full name, price, short bio, and pet sitter rating.
+
+</details>
+
+<details>
+<summary> CLICK HERE</summary>
+
+<p>
+
+![list of pet sitters](https://github.com/ios-course/ironfoudation-team-project/assets/120586844/a62ea90f-50f1-4c44-81c1-e0ef8091feae)
+
+</p>
+</details>
+
+`Get detailed information about the pet sitter`: The app allows users to get more information on the tap of the pet sitter's row in the list like the pet sitter's phone number, rating, and full bio.
+
+</details>
+
+<details>
+<summary> CLICK HERE</summary>
+
+<p>
+
+![sitter](https://github.com/ios-course/ironfoudation-team-project/assets/120586844/4994d3ef-8b12-4083-8924-1730c9b53ef6)
+
+</p>
+</details>
+
+### Notes
```

Comment:
Why do we need this info here?

Reply:
Yes, this info information doesn't apply to this file. I deleted it.